solutions/community-deploy/cloud/AWS/setup.sh

```diff
@@ -30,7 +30,7 @@ for nic in eth1 eth2
 do
    MAC=$(ifconfig $nic | grep -o -E '([[:xdigit:]]{2}:){5}[[:xdigit:]]{2}')
    cp /etc/sysconfig/network-scripts/ifcfg-eth0 /etc/sysconfig/network-scripts/ifcfg-$nic
-   sed -i 's/eth0/$nic/g' /etc/sysconfig/network-scripts/ifcfg-$nic
+   sed -i "s/eth0/$nic/g" /etc/sysconfig/network-scripts/ifcfg-$nic
```

Comment:
Please check that scripts works with this change

Reply:
Yes this is working.